src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useMemo, useState } from 'react';
```

Comment:
EPEL checkbox should be checked and disabled on the Repositories step when some of its package groups is selected on the Packages step as it works for packages now

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/2e22c00a-4117-47df-8cf5-2514b2d0d055)